packages/jaeger-ui/src/components/TraceDiff/TraceDiffHeader/CohortTable.js

```diff
@@ -128,6 +129,13 @@ export default class CohortTable extends React.PureComponent<Props> {
           render={(value, record) => record.state === fetchedState.DONE && formatDuration(value)}
         />
         <Column title="Spans" dataIndex="data.spans.length" key="address" />
+        <Column
```

Comment:
Can you fix the `key` field on the columns being apparently copy-pasta (lastname, address, etc)?

Reply:
updated!